### PR TITLE
Bug 1244267 - Add b2g-ota repo to cached repo list

### PR DESCRIPTION
**Pull request by gregarndt**

### CODE REVIEW

**Comment by selenamarie** [comment]
:+1:

File: src/cli/moz-cache.js [hunk 1]
```diff
@@ -52,6 +52,8 @@ export default async function main(config, argv) {
         'https://hg.mozilla.org/releases/mozilla-b2g37_v2_2r/',
         'https://hg.mozilla.org/releases/mozilla-b2g44_v2_5',
         'https://hg.mozilla.org/releases/mozilla-b2g44_v2_5/'
```
Comment (selenamarie) [inline src/cli/moz-cache.js]: oops needs a comma!

File: src/cli/moz-cache.js [hunk 2]
```diff
@@ -52,6 +52,8 @@ export default async function main(config, argv) {
...
+        'https://hg.mozilla.org/releases/b2g-ota/',
+        'https://hg.mozilla.org/releases/b2g-ota',
```
Comment (selenamarie) [inline src/cli/moz-cache.js]: and no comma there!

Comment (gregarndt) [inline src/cli/moz-cache.js]: pesky commas!

**Pull request merged**